Assignment/solution_student/index.html

```diff
@@ -4,30 +4,82 @@
     <meta charset="UTF-8">
     <title>Relaxr Blog</title>
     <meta name="description" content="An HTML Starter Template">
-    
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,600,800' rel='stylesheet' type='text/css'>  
     <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
     <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
 
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
   <body>
-
-    <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
     
       <header>
-        <h1>FEWD HTML Template</h1>
+          <h1 class="logo">Relaxr</h1>
+          <nav>
+            <ul>
+              <li><a href="#">About</a></li>
+              <li><a href="#">FAQ</a></li>
+              <li><a href="#">Team</a></li>
+              <li><a href="#">Contact Us</a></li>
+              <li><a href="#">Blog</a></li>
+            </ul>
+          </nav>  
       </header>
 
       <main>
-        <p>Our HTML starter template!</p>
+        
+          <div id="main">
+            <article>
```

Comment:
You have used the `<article>` element almost perfectly here, though there are two blog posts encased in a single `article` tag, though they are two distinct articles, so I'd recommend wrapping each blog post in its own `<article>` rather than one big one.